portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFunFact() {
+  const funFacts =
+      ['Favorite TV show is Game of Thrones', 'Favorite Sport is Football', 'I have fractured every finger on my left hand', 'I have played the double bass since Third Grade'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const funFact = funFacts[Math.floor(Math.random() * funFacts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factContainer = document.getElementById('greeting-container');
```

Comment:
You may want to change the name of this containing element (maybe 'fun-fact-container' or similar).